src/vis/bar/utils.ts

```diff
@@ -0,0 +1,240 @@
+import {merge} from 'lodash';
+import {CategoricalColumn, ColumnInfo, EColumnTypes, ESupportedPlotlyVis, IVisConfig, NumericalColumn, Scales} from '../interfaces';
+import {PlotlyInfo, PlotlyData} from '../interfaces';
+import {getCol} from '../sidebar/utils';
+
+export enum EBarDisplayType {
+    DEFAULT = 'Default',
+    NORMALIZED = 'Normalized',
```

Comment:
Is the value of the enum property used as label for the select? If yes, we would need to extract it as language string and use i18n.

Reply:
Since I imagine the way we do this will change in the near future, leaving any implementation of i18n to future PRs.